.github/workflows/diff_prs.yml

```diff
@@ -0,0 +1,53 @@
+name: Diff as PRs
+
+on:
+  workflow_dispatch:
+    inputs:
+      base_branch:
+        description: 'Base branch to compare from (default: develop).'
+        default: 'develop'
+        required: false
+      target_branch:
+        description: 'Target branch to compare to.'
+        required: true
+
+permissions:
+  contents: read
+
+jobs:
+  fetch-prs:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
```

Comment:
Is it a shallow checkout by default ?

Reply:
I have no idea, the doc don't seem to mention it
https://github.com/actions/checkout
What difference does this make except for the execution time and resource used?

Reply:
Here is the relevant info in their doc and it's the good default. the diff is it's not gonna load the entire git history

  # Number of commits to fetch. 0 indicates all history for all branches and tags.
    # Default: 1
    fetch-depth: ''

Reply:
> ```
># Default: 1```
We should be fine by default then :slightly_smiling_face: